web-ui/src/main/resources/catalog/views/default/templates/home.html

```diff
@@ -34,6 +34,81 @@ <h1 data-translate="">getstarted</h1>
       </div>
     </div>
   </div>
+    <div class="row">
+    <div class="col-sm-8" data-ng-show="browse !== ''">
+      <span data-translate="">browseBy</span>
+      <span data-ng-show="searchInfo.facet['inspireThemes'].length > 0 && searchInfo.facet['topicCats'].length > 0">
+      <label data-ng-show="searchInfo.facet['inspireThemes'].length > 0">
+            <input type="radio" name="browse" value="inspire" data-ng-model="browse"/>
+            <span data-translate="">inspireThemes</span>
+          </label>
+          <label data-ng-show="searchInfo.facet['topicCats'].length > 0">
+            <input type="radio" name="browse" value="topics" data-ng-model="browse"/>
+            <span data-translate="">topics</span>
+          </label>
+        </span>
+        <!--button type="button" class="btn btn-default" data-ng-show="searchInfo.facet['inspireThemes'].length > 0" value="inspire" data-ng-model="browse"><span data-translate="">inspireThemes</span></button>
+        <button type="button" class="btn btn-default" data-ng-show="searchInfo.facet['topicCats'].length > 0" value="topics" data-ng-model="browse"><span data-translate="">topics</span></button-->
+      
+      <label data-ng-show="searchInfo.facet['inspireThemes'].length > 0 && searchInfo.facet['topicCats'].length == 0"><span data-translate="">inspireThemes</span></label>
+      <label data-ng-show="searchInfo.facet['topicCats'].length > 0 && searchInfo.facet['inspireThemes'].length == 0"><span data-translate="">topics</span></label>
+      <div class="row">
+        <span id="chips-card" data-ng-repeat="(key, facet) in searchInfo.facet['topicCats']"
+        class="col-xs-6 col-sm-4 col-lg-3" data-ng-show="browse === 'topics'">
+          <div class="badge-icon">
+            <a data-ng-href="#/search?facet.q=topicCat%2F{{facet['@name']}}">
+              <span class="badge-result badge-result-topic" >
+                <i class="fa fa-3x fa-table gn-icon gn-icon-{{facet['@name']}}"></i>
+              </span>
+            </a>
+          </div>
+          <div class="badge-text">
+            <a title="{{facet['@label']}}" data-ng-href="#/search?facet.q=topicCat%2F{{facet['@name']}}">
```

Comment:
This is definitively much better than only using lucene field.
With that change, the facet is selected and the user know that a filter is active. :+1: